### PR TITLE
docs: additional doc touch ups for jsdoc

### PR DESCRIPTION
**Pull request by SlayerOrnstein**
### What did you fix? 
Add `/** @module */` so that jsdoc can detect classes and update the README a bit

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **No**
- Does this introduce opinionated data formatting or manual data entry? **Yes**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **Yes**
- Have I run the linter? **Yes**
- Is is a bug fix, feature request, or enhancement? **Docs**

### CODE REVIEW

File: src/Ability.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
-/**
- * Represents an ability used by the player
- */
+/** @module */
+
```
Comment (TobiTenno) [inline src/Ability.js:2]: don't we want both of these?

Comment (SlayerOrnstein) [reply src/Ability.js]: yes

File: src/Ability.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 /** @module */
 
+/**
+ * Represents a players used ability
+ */
```
Comment (TobiTenno) [inline src/Ability.js:5]: combine please

Comment (SlayerOrnstein) [reply src/Ability.js]: done all except for utils in case we add more functions in there later on

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**